sample-lnd.conf

```diff
@@ -1483,13 +1483,17 @@
 ; db.no-rev-log-amt-data=false
 
 ; If set to true, native SQL will be used instead of KV emulation for tables
-; that support it already. Note: this is an experimental feature, use at your
-; own risk.
+; that support it already (currently this is the invoices DB).
```

Comment:
Maybe do it like this, so it will be easier to enhance when payments are sql as well:
```
; Subsystems which support native SQL tables:
; - Invoices
```